.gitignore

```diff
@@ -0,0 +1 @@
+inventory
```

Comment:
@gbraad Removing `.gitignore` right now from the commit. Will work on the idea when we have all the playbooks ready to run from only one playbook.